src/data/resolvers/mutations/customers.js

```diff
@@ -39,6 +39,14 @@ const customerMutations = {
 
     return company;
   },
+
+  /**
+   * Update customer Companies
+   * @return {Promise} customer object
+   */
+  async customersEditCompanies(root, { _id, ...doc }) {
```

Comment:
What is doc ?. Use explicit variable name like companyIds. Where is this arguments description in function comment section ?